src/PKSim.Core/Services/ParameterUpdater.cs

```diff
@@ -121,6 +121,9 @@ private ICommand resetParameterValue(IParameter targetParameter)
 
       private ICommand setParameterValue(IParameter sourceParameter, IParameter targetParameter, bool forceUpdate)
       {
+         //If the parameter we are updating from is a default parameter, the target parameter should either remain as is (default or not default)
+         bool shouldUpdateDefaultState = !sourceParameter.IsDefault;
```

Comment:
The source parameter form the update is a default parmaeter. Updating the target parameter form the source parameter should not update the default state in the target parameter to false. Instead we decide here if we need to update